libcontainerd/container_windows.go

```diff
@@ -41,6 +41,14 @@ func (ctr *container) start() error {
 		return err
 	}
 
+	if ctr.ociSpec.Windows.Servicing {
+		if err := hcsshim.ShutdownComputeSystem(ctr.containerID, hcsshim.TimeoutInfinite, ""); err != nil {
```

Comment:
I don't like infinite here. Should it be a timeout and subsequent call to TerminateComputeSystem?